### ISSUE TITLE
I am not sure how to run the game

### ISSUE THREAD

**Issue by bahadirery** [mushuacan/DGD203-FirstProject#1]
It would be nice to have quick start section in readme to follow steps (exact commands for users to copy-paste) to start the game.

**Comment by mushuacan** [comment]
Ok, I updated the read me file, can you check it?

**Comment by bahadirery** [comment]
No, I tried all of the methods. The first method is just a shortcut it will not work. For the second method, I clicked the exe file and nothing happened. As for the third, I do not know how to open .sln files. 

Also, I did not get which requirements I should install. It would be nice to have more specific instructions with links. If possible, specific commands that would work in terminal would be nice. For example as a start:

`git clone https://github.com/mushuacan/Kervan.git `

and

`cd Kervan`

as for the requirements, maybe `winget` commands would be sufficient.

**Comment by mushuacan** [comment]
I find the problem, I think. For running the program it requires .net 8.0 so I added a link for downloading it.

I think it is playable now, thanks for feedback.
